### PR TITLE
fix: minor issues

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

This PR addressed the following minor issues in the repo:
- Removed unused `.version` file.
- Added `.nvmrc` file.
- Updated the latest setup command in the readme file.
- Updated `sample.env` to the latest.
- Made the application's port number read from an environment variable.
- Fixed issue with `NUM_PROCS` env variable (string to int conversion).
- Added error handling to `bugsnag.js`.

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/versionedRouter.js [hunk 1]
```diff
@@ -28,8 +28,6 @@ const { TRANSFORMER_METRIC } = require("./v0/util/constant");
 const networkHandlerFactory = require("./adapters/networkHandlerFactory");
 const profilingRouter = require("./routes/profiling");
 const destProxyRoutes = require("./routes/destinationProxy");
-
-require("dotenv").config();
```
Comment (koladilip) [inline src/versionedRouter.js:32]: not required now?

Comment (saikumarrs) [reply src/versionedRouter.js:32]: It's already imported [here](https://github.com/rudderlabs/rudder-transformer/blob/dff00b206615bb174863c9ad3eb3e7a879b34b2e/src/index.js#L4) in the entry point script.

File: sample.env [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
-# Marketo specific env variables
-################################################
-# in seconds
-MARKETO_LEAD_CACHE_TTL=10
-MARKETO_AUTH_CACHE_TTL=10
\ No newline at end of file
+PORT=9090 # application's port
+ENABLE_FUNCTIONS=true # user transformation functions
+CLUSTER_ENABLED=true # cluster mode
+NUM_PROCS=1 # Number of workers in the cluster mode
```
Comment (utsabc) [inline sample.env:4]: Why are running transformer in cluster mode in such case?

**Review by krishna2020 — APPROVED** [review]

File: sample.env (hunk 2 above)
Comment (saikumarrs) [reply sample.env:4]: It's similar to the production environment configuration.

Comment (saikumarrs) [reply sample.env:4]: But we need to validate it for sure.

**Review by koladilip — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**